spritz/README.md

```diff
@@ -0,0 +1,153 @@
+[our blog post "Whole Lottie Love"](https://www.novoda.com/blog/whole-lottie-love/), which contains basic instructions on how to export an animation
+into a Lottie-supported format.
+
+Your animation should have:
+
+- for each page, a "swipe away" animation
+- for each page, optionally, an "autoplay" animation segment
+
+These segments must be consecutive to one another, as if the user continuously moves to one page to another.
+
+### Code!
+
+After getting your `LottieAnimationView` and your `ViewPager`, just create a `Spritz` object with the following syntax:
+
+```
+Spritz
+    .with(lottieAnimationView)
+    .withDefaultSwipeAnimationDuration(300, TimeUnit.MILLISECONDS)
+    .withDefaultSwipeForwardInterpolator(SWIPE_FORWARD_INTERPOLATOR)
+    .withPages(
+            new SpritzPage.Builder()
+                    .withAutoPlayDuration(500, TimeUnit.MILLISECONDS)
+                    .withSwipeDuration(500, TimeUnit.MILLISECONDS)
+                    .build(),
+            new SpritzPage.Builder()
+                    .withAutoPlayDuration(500, TimeUnit.MILLISECONDS)
+                    .withSwipeDuration(500, TimeUnit.MILLISECONDS)
+                    .withSwipeBackwardsInterpolator(SWIPE_BACKWARDS_INTERPOLATOR)
+                    .build(),
+            new SpritzPage.Builder()
+                    .withAutoPlayDuration(500, TimeUnit.MILLISECONDS)
+                    .build()
+    )
+    .attachTo(viewPager);
+```
+
+## Technical documentation
+
+### Make your own kind of Spritz
+
+Start building your view pager animation by calling:
+
+```
+SpritzPage.with(lottieAnimationView)
+```
+
+This will set the specific `LottieAnimationView` as the target of your animations. The Lottie view must load the animation separately, as it is not 
+in the scope of this library.
+
+After that, you can proceed setting your view pager transitions.
+
+#### `withDefaultSwipeAnimationDuration`
+
+`withDefaultSwipeAnimationDuration(long duration, TimeUnit timeUnit)` allows you to specify how long the swiping animation has to complete if the user 
+stops dragging.
+
+#### `withDefaultSwipeForwardInterpolator`
+
+`withDefaultSwipeForwardInterpolator(TimeInterpolator swipeForwardInterpolator)` sets the interpolator to use for all pages to complete the swipe 
+animation if the user stops dragging forwards at any point (before starting the autoplay animation, if any). If you don't set this, it will default to
+`LinearInterpolator`. You can always override this default for single pages (see [`withSwipeForwardInterpolator`](#withswipeforwardinterpolator)).
+
+#### `withDefaultSwipeBackwardsInterpolator`
+
+`withDefaultSwipeBackwardsInterpolator(TimeInterpolator swipeBackwardsInterpolator)` sets the interpolator to use for all pages to complete the swipe 
+animation if the user stops  dragging backwards at any point. If you don't set this, it will default to `LinearInterpolator`. You can always override 
```

Comment:
Double space: `stops  dragging`

Reply:
Done in 30085d03d38ca15e25083f45da022d506a92ad45.